testing-project/postgres/dbt_project.yml

```diff
@@ -0,0 +1,34 @@
+# Name your project! Project names should contain only lowercase characters
```

Comment:
This testing-project will have to be removed before merging to main. Why is it checked in here?